controllers/user.js

```diff
@@ -107,6 +110,27 @@ class Users {
     */
   static async register(req, res) {
     const { username, email, password } = req.body;
+    sgMail.send({
+      to: email,
+      from: 'noreply@learncode.academy',
+      subject: 'Verification Email from LearnGround',
+      html: `<h2 style=" background-color:blue;color:white;margin-left:30px;
+      padding:15px">
+      Welcome To LearnGround</h2><br>
+       <strong style="color:green;">
+       Your Registration was sucessfull </strong><br>
+       <strong style="color:green;margin-left:30px;">
+       To activate Your account click the link below </strong><br>
+       <em style="color:blue;margin-left:30px;">
+       https://authors-haaven.herokuapp.com/ </em>`
+    }, (error) => {
+      if (error) {
```

Comment:
What is the actual/exact error that can be gotten here? It is that error we need to find and send to the error handler. Logging the error here is not useful to the application user, only to the developer.

Reply:
ok... will take care of this